views/forms/ChartFilterForm/ChartFilterForm.css

```diff
@@ -3,7 +3,7 @@
   width: 100%;
   gap: 10px;
   padding-bottom: 20px;
-  grid-template-columns: repeat(1fr, 4);
+  grid-template-columns: repeat(1fr, 5);
```

Comment:
I wonder if we should also add a maxWidth, the client expressed a dislike when the filters expand wider than the charts..